adafruit_ads1x15/ads1x15.py

```diff
@@ -15,11 +15,13 @@
 __repo__ = "https://github.com/adafruit/Adafruit_CircuitPython_ADS1x15.git"
 
 import time
-from micropython import const
+
 from adafruit_bus_device.i2c_device import I2CDevice
+from micropython import const
 
 try:
     from typing import Optional
+
```

Comment:
Are these new lines needed?

Reply:
No. I think it was w.e pylint or pep linter i have installed. I can remove it.

Reply:
Can someone explain this to me https://github.com/adafruit/Adafruit_CircuitPython_ADS1x15/pull/86/files#diff-ad1a84f70997c40a92f1b3d7688f0cbf2311c39d2e453f21b2c779f0f74d13c7R37

What is meant here? Gains index's are type float or int?

Reply:
The `gains` property returns a list of `float` since one of them is a fraction.